src/sentry/api/endpoints/project_rule_preview.py

```diff
@@ -58,4 +68,8 @@ def post(self, request: Request, project) -> Response:
             on_results=lambda x: serialize(x, request.user),
             count_hits=True,
         )
+        response.data = {
+            "data": response.data,
+            "endpoint": data["endpoint"],
```

Comment:
is it worth maybe wrapping this in `metadata` dict in case we want to add more to it? trying to find some similar examples in the codebase

Reply:
Maybe we could even just add it as a header, it would make this backwards compatible this way too

Reply:
probably even better. can talk about this during meeting or you can just change it if you find a good precedent